### ISSUE TITLE
0.8.4 doesn't respect moment.js first day of the week

### ISSUE THREAD

**Issue by ravicious** [gpbl/react-day-picker#107]
Hi,

We're using react-day-picker 0.8.4 (the same bug is present in 0.8.3 though). Our moment.js locale settings look like this:

```
  moment.locale('en', {
    week: {
      dow: 1
    }
  });
```

If I open the day picker for December 2015, it tells me 3.12.2015 is Wednesday, even though it's Thursday.

![image](https://cloud.githubusercontent.com/assets/27113/11556652/66b7d542-99a8-11e5-9f3a-472394cea709.png)

If I drop the moment.js `dow` setting, everything's fine with the day picker, but then the rest of our code fails as it expects the first day of the week to be Monday.

(so they're exactly the same as in https://github.com/gpbl/react-day-picker/issues/26#issuecomment-107103753)

We're in CEST right now, I think this bug was not present during CET.

I tried upgrading the lib to 1.1.5, but:

  * It won't work with React 0.13.3 (React complains `<DayPicker />` doesn't return a valid React component.
  * It drops dependency on moment.js, which we use heavily in our app. I'd rather not depend on the built-in Date class. Our app has to work in weird SmartTV browsers and I don't trust their JS implementations. ;)
  * It completely changes the HTML nodes and styles, requiring us to rewrite our styles.

I created this issue, because I've seen there have been more bugs with DST fixed in > 1.0 versions, but there's no migration path and other issues like React 0.13.3 complaining about DayPicker. :( We'd appreciate having a patch we could apply.

**Comment by JKillian** [comment]
FYI @ravicious, the latest release, [`1.2.0`](https://github.com/gpbl/react-day-picker/releases/tag/v1.2.0) should fix things in React 0.13.

**Comment by ravicious** [comment]
Alright, I'll take a look at it later. Thanks! 🙏
